.vscode/tasks.json

```diff
@@ -1,43 +1,52 @@
 {
   // See https://go.microsoft.com/fwlink/?LinkId=733558
   // for the documentation about the tasks.json format
-  "version": "0.1.0",
+  "version": "2.0.0",
   "tasks": [
     {
-      "taskName": "Install all dependencies",
-      "command": "./node_modules/.bin/lerna",
-      "isShellCommand": true,
+      "label": "Install all dependencies",
+      "group": "build",
+      "command": "npm",
+      "type": "shell",
       "args": [
+        "run",
+        "-s",
         "bootstrap"
-      ]
+      ],
+      "problemMatcher": []
     },
     {
-      "taskName": "Remove all dependencies",
-      "command": "./node_modules/.bin/lerna",
-      "isShellCommand": true,
+      "label": "Remove all dependencies",
+      "group": "build",
+      "command": "npm",
+      "type": "shell",
       "args": [
-        "clean",
-        "--yes"
-      ]
+        "run",
+        "-s",
+        "clean:lerna"
+      ],
+      "problemMatcher": []
     },
     {
-      "taskName": "Run all tests",
+      "label": "Build, test and lint",
+      "group": "test",
       "command": "npm",
-      "isShellCommand": true,
+      "type": "shell",
       "args": [
         "run",
-        "test",
-        "-s"
+        "-s",
+        "test"
```

Comment:
Is this running https://github.com/strongloop/loopback-next/blob/master/package.json#L40? I feel like from the title it would run https://github.com/strongloop/loopback-next/blob/master/package.json#L38.

Reply:
Ah, good catch. This task should execute `npm test -s`. It's called "build, test and lint" because that's what `npm test` does under the hood (see `pretest` and `posttest` scripts). I'll fix it in a moment.

This is the task you will run during your red-green-refactor cycle. You don't want to run the full rebuild including reinstalling dependencies whenever you write a new test or make a failing test pass!

Reply:
oh I see, that's great. Thank you for explaining! Definitely do not want to run full rebuild haha.